src/tools/react-native.ts

```diff
@@ -68,17 +67,10 @@ export const getReactNativeConfig = tool({
   parameters: z.object({}),
   execute: async () => {
     try {
-      const {
-        root,
-        reactNativePath: path,
-        reactNativeVersion: version,
-        project,
-        platforms,
-      } = await loadReactNativeConfig()
+      const { root, reactNativePath: path, project, platforms } = await loadReactNativeConfig()
```

Comment:
Turns out React Native CLI is lying with versions, it's better for AI to figure it out based on package.json

![image](https://github.com/user-attachments/assets/45b5ed35-0ad0-4c53-ae60-f5a00d6f7441)